x-pack/plugins/cloud_security_posture/common/schemas/csp_rule_template.ts

```diff
@@ -7,17 +7,18 @@
 import { schema as rt, TypeOf } from '@kbn/config-schema';
 
 const cspRuleTemplateSchema = rt.object({
+  id: rt.string(),
   name: rt.string(),
+  tags: rt.arrayOf(rt.string()),
   description: rt.string(),
   rationale: rt.string(),
-  impact: rt.string(),
   default_value: rt.string(),
+  impact: rt.string(),
   remediation: rt.string(),
   benchmark: rt.object({ name: rt.string(), version: rt.string() }),
-  severity: rt.string(),
-  benchmark_rule_id: rt.string(),
   rego_rule_id: rt.string(),
-  tags: rt.arrayOf(rt.string()),
+  enabled: rt.boolean(),
```

Comment:
maybe rename to `enabled_default` and `muted_default` if this is the idea.

Reply:
we took that from your examples [here](https://github.com/elastic/package-spec/pull/276/files#r803648725)
would that mean to change the spec as well? if so, lets delay that since it is out of scope from this pr

Reply:
doesn't require a spec change to merge.
optimally we should update the spec examples later with real examples also including our markdown fields, etc..
Renaming is not mandatory anyways.